src/components/admin/translation-controls/TranslationControls.module.scss

```
@@ -17,21 +17,21 @@
 .language-select-head {
     display: inline-flex;
     align-items: center;
-    
+
     border: none;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)